src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/Ordering/OrderingHelper.cs

```diff
@@ -33,20 +33,14 @@ public static Task MoveAsync(ConfiguredProject configuredProject, IProjectAccess
                 project.ReevaluateIfNecessary();
                 ImmutableArray<ProjectItemElement> addedElements = GetAddedItemElements(previousIncludes, project);
 
-                switch (action)
+                // TODO: Should the result (success or failure) be ignored?
+                bool _ = action switch
```

Comment:
Discards like this don't need a type declaration.

```suggestion
                _ = action switch
```